package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-chime-sdk-js",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
@aburnson @vidya-mohan
I had the same question in another PR. Are we patching per PR? (Sorry, if I'm missing something.)

Reply:
Good call out. Need some clarifications.

Reply:
@simmkyu @XHatan 
Yes, we are bumping patch on a per-commit basis. This guarantees that each commit to master receives a unique version number. We adopted this approach in order to track version adoption down to the commit level and to make it easier to determine which version canaries are running (the version number is visible in the lower right corner of the meeting demo now).